### DISCUSSION TITLE
Search API docs

### DISCUSSION THREAD

**Discussion by shaedrich** [community/community#69806] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

I'm either not seeing it or the docs are currently missing a way to filter only the API docs. Which, to me, is quite annoying since I ofttimes can't find what I'm looking for by using the search. So in case there is no way to do so yet, I recommend:
* Either scoping the search input field to the section of the docs ("REST API" in our case)
* Adding section filters to the search result page

**Comment by ghostinhershell** [comment]
Hi @shaedrich, 

Thanks so much for submitting this feedback. Just jumping in to let you know that I've shared this feedback with our Docs team so that it's on their radar. If you have any other questions or feedback to share, feel free to follow-up and let me know. Cheers!

**Reply by shaedrich** [reply]
Thank you, I will!
